### PR TITLE
Add slfo-packagelist-uploader and a adjusted check_source

### PR DESCRIPTION
**Pull request by nilxam**

### CODE REVIEW

**Comment by nilxam** [comment]
I don't get the smoketest error:

`
import git
ModuleNotFoundError: No module named 'git'
Error: Process completed with exit code 1.
`

python3-GitPython is added, what should I do here?

File: check_source.py [hunk 1]
```diff
@@ -147,11 +150,22 @@ def check_source_submission(
             self.review_messages['declined'] = f'May not modify a non-source package of type {kind}'
             return False
 
-        if not self.allow_source_in_sle and self.sle_project_to_check:
-            if entity_exists(self.apiurl, self.sle_project_to_check, target_package):
+        if not self.allow_source_in_sle:
+            if self.sle_project_to_check and entity_exists(self.apiurl, self.sle_project_to_check, target_package):
                 self.review_messages['declined'] = ("SLE-base package, please submit to the corresponding SLE project."
                                                     "Or let us know the reason why needs to rebuild SLE-base package.")
```
Comment (lkocman) [inline check_source.py:156]:
```suggestion
                self.review_messages['declined'] = ("SLFO package, please submit change to the corresponding project."
                                                    "Alternatively please provide a reason for rebuilding this SLFO package.")
```

Comment (lkocman) [reply check_source.py:156]: I think we could use SLFO as the package could be from SLES, SL Micro or similar.  Not sure if it helps the maintainer as SLFO is not exactly common term in community. Alternatively we could say just SLES as SLED is going away anyway.

**Review by lkocman — APPROVED** [review]
Generally aprooving, see my comment about the text

**Review by lkocman — APPROVED** [review]
looks good! thank you @nilxam

**Pull request merged**